utils/near-performance-metrics/src/stats_enabled.rs

```diff
@@ -168,6 +181,34 @@ pub(crate) fn get_entry() -> Arc<Mutex<ThreadStats>> {
     LOCAL_STATS.with(|x| (*x).clone())
 }
 
+#[cfg(target_os = "linux")]
+fn get_c_memory_usage_cur_thread() -> ByteSize {
```

Comment:
It seems that with this change, if we run neard with `--performance_stats` without the special way of starting it described in the comment, then it would crash, which I don't think is a good idea

Reply:
This code wouldn't crash, malloc is guaranteed to return NULL when allocating given chunks of memory larger than can be allocated.

Reply:
I added a feature "c_memory_stats" just in case.

Reply:
Yeah, naively I'd expect a call do a different symbol. But we need [weak linkage](https://en.wikipedia.org/wiki/Weak_symbol) for that, and it's not available in rust at the moment.